### PR TITLE
add vshpere vcenter-1 job

### PR DESCRIPTION
**Pull request by qiliRedHat**

### CODE REVIEW

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install

**Comment by qiliRedHat** [comment]
Test [job](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/58067/rehearse-58067-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install/1848741847552233472) failed for No such file or directory when building local variables and platform spec for pool vcenter-1.devqe.ibmc.devcluster.openshift.com-devqedatacenter-1-devqecluster-1 on line [386]
```
(https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/vsphere/check/vcm/ipi-conf-vsphere-check-vcm-commands.sh#L386)
2024-10-22 15:06:02+00:00 -  + building local variables and platform spec for pool vcenter-1.devqe.ibmc.devcluster.openshift.com-devqedatacenter-1-devqecluster-1
/bin/bash: line 386: null: No such file or directory
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"sigs.k8s.io/prow/pkg/entrypoint/run.go:84","func":"sigs.k8s.io/prow/pkg/entrypoint.Options.internalRun","level":"error","msg":"Error executing test process","severity":"error","time":"2024-10-22T15:06:02Z"}
error: failed to execute wrapped command: exit status 1 
INFO[2024-10-22T15:06:03Z] Step ingress-perf-ipv4ipv6dual-install-ipi-conf-vsphere-check-vcm failed after 6s. 
INFO[2024-10-22T15:06:03Z] Step phase pre failed after 12s.
```
There is no annotations under metadata
`VCENTER_AUTH_PATH=$(jq -r '.metadata.annotations["ci-auth-path"]' < /tmp/pool.json)`

**Comment by qiliRedHat** [comment]
Test [job](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/58067/rehearse-58067-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install/1849635944542507008) failed for `tls: failed to verify certificate: x509: certificate signed by unknown authority`
```
2024-10-25 02:18:10+00:00 -  + Creating vsphere_context.sh file...
2024-10-25 02:18:10+00:00 - Find virtual machines attached to ci-vlan-1165 in DC /devqedatacenter-1 and destroy
govc: Post "https://vcenter-1.devqe.ibmc.devcluster.openshift.com/sdk": tls: failed to verify certificate: x509: certificate signed by unknown authority
2024-10-25 02:18:10+00:00 -  + checking /tmp/secret/LEASE_vsphere-elastic-46-wspb4.json and ci-vlan-1[165](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/58067/rehearse-58067-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install/1849635944542507008#1:build-log.txt%3A165) for DVS UUID
2024-10-25 02:18:10+00:00 -  + gathering clusters and UUIDs associated with //devqedatacenter-1/network/ci-vlan-1165
govc: Post "https://vcenter-1.devqe.ibmc.devcluster.openshift.com/sdk": tls: failed to verify certificate: x509: certificate signed by unknown authority
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"sigs.k8s.io/prow/pkg/entrypoint/run.go:84","func":"sigs.k8s.io/prow/pkg/entrypoint.Options.internalRun","level":"error","msg":"Error executing test process","severity":"error","time":"2024-10-25T02:18:10Z"}
error: failed to execute wrapped command: exit status 1 
INFO[2024-10-25T02:18:11Z] Step ingress-perf-ipv4ipv6dual-install-ipi-conf-vsphere-check-vcm failed after 6s.
```

**Comment by rvanderp3** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install

**Comment by rvanderp3** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ingress-perf-ipv4ipv6dual-install

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ipv4ipv6dual-control-plane-3nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ipv4ipv6dual-control-plane-24nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ipv4ipv6dual-control-plane-24nodes

**Review by paigerube14 — APPROVED** [review]
/lgtm

File: ci-operator/step-registry/ipi/conf/vsphere/check/vcm/ipi-conf-vsphere-check-vcm-ref.yaml
Comment (paigerube14) [inline ci-operator/step-registry/ipi/conf/vsphere/check/vcm/ipi-conf-vsphere-check-vcm-ref.yaml:28]: since you're adding this as the default and a parameter, we might need to take out [this line ](https://github.com/openshift/release/blob/96444bb30c9b2e40da5091181f75759e0c8b91f6/ci-operator/step-registry/ipi/conf/vsphere/check/vcm/ipi-conf-vsphere-check-vcm-commands.sh#L35)

Comment (qiliRedHat) [reply ci-operator/step-registry/ipi/conf/vsphere/check/vcm/ipi-conf-vsphere-check-vcm-ref.yaml:28]: @paigerube14  Good cache! Thanks

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ipv4ipv6dual-control-plane-3nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ipv4ipv6dual-control-plane-3nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ipv4ipv6dual-control-plane-24nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-ipv4ipv6dual-control-plane-24nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-vcenter-1-ipv4ipv6dual-control-plane-3nodes

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml
Comment (paigerube14) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:9650]: lets add the slack reporter here

**Comment by qiliRedHat** [comment]
/pj-rehearse  periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-vcenter-1-ipv4ipv6dual-control-plane-24nodes

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml
Comment (qiliRedHat) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:11494]: Thanks. Added.

**Comment by qiliRedHat** [comment]
@jcpowermac Would you help to review? I need an approval from vsphere-approvers.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__vsphere-4.18-nightly-x86.yaml
Comment (jcpowermac) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__vsphere-4.18-nightly-x86.yaml:139]: is this kinda excessive? are you notifying the users of vcenter-1 devqe for each run?

Comment (rvanderp3) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__vsphere-4.18-nightly-x86.yaml:139]: be sure to define the number of cores and memory in GB you'll need for each job.  eventually, we may start enforcing resource leasing and constraints in devqe to ensure all users of the environment are able to run their tests.

https://github.com/openshift/release/blob/99e85f00df4aceed6f7d18bfc6c937f40ac91667/ci-operator/step-registry/ipi/conf/vsphere/check/vcm/ipi-conf-vsphere-check-vcm-ref.yaml#L19-L25

Comment (qiliRedHat) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__vsphere-4.18-nightly-x86.yaml:139]: @jcpowermac 
> is this kinda excessive?
Once a week is not very frequent for CI. We will have another job in the future, at that time, I can make this twice a month and the new one twice a month.

> are you notifying the users of vcenter-1 devqe for each run?

No. I'm not aware of the rest users of vcenter-1 devqe. When I requested this env Jim said "we could set up a new vCenter to manage that 2 node cluster and configure it for ipv6. This would also not impact the current DEVQE set up and resources (which are over half for cpu and memory for the cluster).". So I supposed the rest users are on the previous envs. https://issues.redhat.com/browse/DPP-14784?focusedId=24737745&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24737745. 
I can see some other vms on the vcenter console, 3 of them have pewang in the vm names, I have noticed him. But I can not tell who is the owner for the rest of them, like the vms with prefix 'nested7' and 'nested8'. Would you suggest what I should do?

Comment (qiliRedHat) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__vsphere-4.18-nightly-x86.yaml:139]: @rvanderp3 Are you sure to have such big size as default? For our team 4 CPU cores 16G memory is enough as a default.

Comment (jcpowermac) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__vsphere-4.18-nightly-x86.yaml:139]: > @rvanderp3 Are you sure to have such big size as default? For our team 4 CPU cores 16G memory is enough as a default.

6 nodes x 4 cpus = 24
6 nodes x 16gb = 96

its the total cores and ram required

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vsphere-4.18-nightly-x86-vcenter-1-ipv4ipv6dual-control-plane-24nodes

**Comment by qiliRedHat** [comment]
@jcpowermac @rvanderp3 , I did the update and the re-test passed. Please help to review and approve if you think it's good. Thanks!

**Pull request merged**
